### PR TITLE
added mongoose web service icon

### PR DESCRIPTION
**Pull request by StokicDusan**
mongoose

**Issue:** #5594

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: _data/simple-icons.json
Comment (StokicDusan) [inline _data/simple-icons.json:5538]: should be 
#696A6C

**Comment by adamrusted** [comment]
Thanks for the contribution @StokicDusan - is there an SVG version you've based your icon on, or is it a trace of the PNG? I've dropped Cesanta an email, asking them to send anything across, and hopefully give their okay on us including the icon once it's done!

**Comment by StokicDusan** [comment]
I believe it is a trace. I got the PNG off of [https://stackshare.io/mongoose-web-server](url) and dropped it in a PNG to SVG converter.  
I edited the logo in Inkscape and SVGOMG

**Comment by adamrusted** [comment]
No worries, Let's wait and see if Cesanta get back to me on whether there is an official Vector file we can use. Looking at the checks on this PR, it'd be worth updating your SVG file's name to `mongoose.svg` and adding the `<title>` tag as detailed in the contribution guidelines.

**Comment by PeterShaggyNoble** [comment]
Mongoose Web Server has its own website at https://mongoose.ws/ and, from that, it looks like [the logo](https://mongoose.ws/images/logo.png) is actually different from what's included in this PR. An SVG can be found at https://github.com/cesanta/mongoose-os-ide/blob/d7bfb0159e9b9c3cdf6f2726e3e3985bd8807b5d/resources/logo.svg

Could you update using that, please, @StokicDusan?

**Comment by StokicDusan** [comment]
It seems you're right ,@PeterShaggyNoble. I'll change it.
@adamrusted, has Cesanta sent you anything?

**Comment by adamrusted** [comment]
@StokicDusan - nothing from Cesanta, but given @PeterShaggyNoble has found the SVG available on their GitHub, I think we're good to use that 👍

**Comment by StokicDusan** [comment]
@adamrusted I don't know how to fix this issue.  
This is my first contribution of this kind so forgive me for not knowing how to approach this problem. If you have any suggestions on how I could fix this, it's welcome.
Otherwise, should I just drop this PR and try to push again with just one commit?

**Comment by adamrusted** [comment]
No problem, @StokicDusan - it looks like you currently have two paths within the SVG. These can be merged in your SVG software using the 'Unite' option, usually in the 'Pathfinder' section. It's also failing as we no longer require the word 'icon' in the `<title>` tag - so if you're able to remove that, and merge `upstream/develop` into your branch - that should fix some of the issues you're having.
If you'd rather I take a look at this path, I can do - but we'd need another maintainer to review / merge it if I've worked on it.

**Comment by StokicDusan** [comment]
It finally passed!!  
I plan to contribute more :rocket:  
Thanks for all the help @adamrusted, @PeterShaggyNoble !!!

**Review by adamrusted — CHANGES_REQUESTED** [review]
Great work @StokicDusan! 💪
The path matches up perfectly - so nice job.

The only thing I'm seeing is a few [points that could be removed](https://user-images.githubusercontent.com/1830867/120288888-9c7ada00-c2b8-11eb-9d4a-666abc477019.png) on straight lines. I wouldn't worry too much about the curves - but you could definitely delete a few points to shrink the file size further. Did you want to look at doing that, or shall I take a quick look at it?

**Comment by StokicDusan** [comment]
Sure, @adamrusted, you can take a look at it and try to shrink the file size down.

**Comment by adamrusted** [comment]
As usual, I went a bit overboard when removing points, so had to return to @StokicDusan's file 😅
@PeterShaggyNoble - please check you're happy with below path before I merge it into this PR. I've tried removing points around the circles, but we end up with a fair bit of distortion - so probably best to keep them as they are for now.

```svg
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Mongoose</title><path d="M12.01 11.246a.968.968 0 0 0-.328.104c-.225.117-.422.416-.422.646 0 .23.197.532.422.645a.908.908 0 0 0 .328.105.908.908 0 0 0 .328-.105c.225-.113.422-.415.422-.645 0-.23-.197-.53-.422-.646a.982.982 0 0 0-.328-.104zm6.383-.047a.986.986 0 0 0-.33.104c-.226.117-.422.416-.422.646a.905.905 0 0 0 .107.328c.113.225.415.424.645.424a.905.905 0 0 0 .328-.107c.226-.113.422-.415.422-.645 0-.23-.196-.53-.422-.646a.982.982 0 0 0-.328-.104zM8.818 8.104a.989.989 0 0 0-.328.103c-.225.117-.424.417-.424.647a.92.92 0 0 0 .11.33c.113.225.412.421.642.421a.914.914 0 0 0 .328-.109c.225-.113.422-.412.422-.642a1.022 1.022 0 0 0-.101-.329c-.117-.225-.419-.421-.649-.421zm6.383-.048a.986.986 0 0 0-.33.104c-.225.117-.422.418-.422.647 0 .23.197.53.422.642a.9.9 0 0 0 .33.11.91.91 0 0 0 .328-.11c.225-.113.422-.412.422-.642 0-.23-.197-.53-.422-.647a1.03 1.03 0 0 0-.328-.104zm.164-.74c.128.014.256.044.385.092v.002c.558.206.951.788.951 1.397 0 .615-.393 1.193-.96 1.404-.23.089-.347.102-.643.088l-.362-.024c-.309.313-.936.942-1.248 1.252l.014.368c.023.492-.103.829-.432 1.162a1.492 1.492 0 0 1-2.12 0c-.968-.97-.255-2.598 1.12-2.555l.32.008c.333-.329 1.002-.996 1.333-1.328l-.016-.319a1.494 1.494 0 0 1 1.658-1.547zM11.514.006c-.198.008-.37.024-.49.049a5.117 5.117 0 0 0-1.971.877c-.498.375-8.007 7.912-8.237 8.273-.4.61-.669 1.301-.773 1.957-.084.525-.037 1.646.094 2.125.127.488.535 1.324.84 1.727.136.173.259.312.277.312.291-.243 4.989-4.957 6.086-6.051l-.02-.336c-.019-.474.114-.82.438-1.144.948-.943 2.562-.273 2.562 1.059 0 .868-.71 1.534-1.582 1.492l-.342-.014c-1.515 1.513-4.564 4.561-6.077 6.075 1.295 1.296 6.149 6.169 6.64 6.585a5.157 5.157 0 0 0 1.783.877c.512.14 1.675.178 2.21.065a4.954 4.954 0 0 0 2.01-.868c.539-.441 5.429-5.348 6.763-6.683-.735-.733-2.214-2.212-2.948-2.946l-.324.008a1.424 1.424 0 0 1-1.123-.435 1.492 1.492 0 0 1 0-2.12c.967-.966 2.62-.244 2.555 1.116l-.016.318 2.967 2.961.184-.25a5.363 5.363 0 0 0 .89-1.865c.07-.281.09-.526.09-1.197 0-.741-.014-.89-.113-1.243a6.64 6.64 0 0 0-.69-1.535c-.206-.319-7.71-7.864-8.183-8.23C14.629.67 13.789.255 13.3.123c-.362-.094-1.193-.14-1.786-.117z"/></svg>
```

**Comment by PeterShaggyNoble** [comment]
Cack! Sorry, didn't notice that the SVG I found is actually Potraced and, therefore, pretty low quality. I think recreating it from scratch, given the simple shapes involved, would probably be the better approach or, at least, we should clean up the circles from the original SVG.

**Comment by adamrusted** [comment]
I'm just about to upload a made-from-scratch version, tracing around the SVG source.
Given it's a badly potraced version - I've taken some liberties, see the source in black and my version in green.
image

**Comment by PeterShaggyNoble** [comment]
How does it line up against [this raster file](https://mongoose.ws/images/logo.png), @adamrusted?

**Comment by ericcornelissen** [comment]
Assuming the SVG in the Pull Request is the most recent, it lines up pretty well with the raster file but I'd say there's room for improvement with regards to the alignment of the two circles on left. @adamrusted, what do you think?

| SVG on top of raster | Raster on top of SVG |
|---|---|
| image | image |

**Comment by adamrusted** [comment]
Superseded by #8913

**Pull request closed without merging**
